src/main/java/com/google/devtools/build/lib/syntax/StringModule.java

```diff
@@ -893,7 +894,13 @@ public Integer count(String self, String sub, Integer start, Object end)
               + "and <code>end</code> being exclusive.",
       parameters = {
         @Param(name = "self", type = String.class, doc = "This string."),
-        @Param(name = "sub", type = String.class, legacyNamed = true,
+        @Param(
+            name = "sub",
+            allowedTypes = {
+              @ParamType(type = String.class),
+              @ParamType(type = SkylarkList.class, generic1 = String.class),
```

Comment:
Python allows only string or tuple-of-string here, not list of string, and nor should we.

Reply:
Done